### ISSUE TITLE
Dota 2 is dying

### ISSUE THREAD

**Issue by Ersultan13** [ValveSoftware/Dota2-Gameplay#3831]
### Description

Hello Valve. and dear IceFrog
My name is Ersultan I am from Kazakhstan.
A big fan of your game has spent more than 14 years in the game since AllStars.

I want to say that the activity of players is falling, and the game is deteriorating, there are many points that need to be improved, to create something new.

1. Rating selection system
2. The system of punishment namely (Single Draft)
3. Balance of heroes
You can write about Dota 2 problems endlessly, but I would like to offer my service.

I want you to take me to work on Dota 2, I'm sure I can keep the community, bring back the past, and bring new players into the game.

I would write a big text about the problems of the game and about myself, but I'm not sure that applications are being considered in a good manner, so if there is an answer, you can contact me for further conversation

ersultan1310@yandex.ru
ers324627@gmail.com
vk.com/e0147182

**Comment by Liam-NFLD** [comment]
What's wrong with the single draft punishment system? How would you improve it?

**Comment by Ersultan13** [comment]
> Что не так с единой системой наказаний на драфте? Как бы вы улучшили его?

I'm not a very frequent guest in low priority, but I noticed that in order to find a game in this mode, I wait on average from 40 minutes to 90 to find a game
And my suggestion would be to introduce time bans as in Cs:GO and reduce decency in a large size. 
 
It took me 3 days to complete 5 games in low priority, considering that 5 wins out of 6 games

**Comment by commentator24** [comment]
time bans does not work because you can just login with another account. and there is already incremental time bans for each game abandoned i think. first one is like 0, then 30 min, then 1 hour then maybe 24h etc. the same for total games i think it start with 1, 3 then 5 LP games or something. maybe someone can confirm for me.

it takes a while to find games because it depends on how many players queuing for single draft mode in that region. just search in other region as well

**Comment by Ersultan13** [comment]
The search for a game in low priority lasts in any case from 30 minutes, even with 3 regions set. And it's unbearably long. And I pointed out, in addition to the time ban, the player's decency fell by more than it is now.

> time bans does not work because you can just login with another account. and there is already incremental time bans for each game abandoned i think. first one is like 0, then 30 min, then 1 hour then maybe 24h etc. the same for total games i think it start with 1, 3 then 5 LP games or something. maybe someone can confirm for me.
> 
> it takes a while to find games because it depends on how many players queuing for single draft mode in that region. just search in other region as well

**Comment by commentator24** [comment]
what do you propose then if time bans does not work?
im also curious what region takes 30 min.

**Comment by Ersultan13** [comment]
> what do you propose then if time bans does not work? im also curious what region takes 30 min.

I propose a reduction of decency in a much larger account. 

And as for the search from 30 minutes, these are 3 included regions. 

Russia, Europe, USA

**Comment by commentator24** [comment]
im not sure what you meant by reducing decency in larger accounts / size..? maybe something wrong in english translation.
reduced penalty in high level accounts? e.g. 1 LP only? this seems unfair though.
reduced threshold / standard? e.g. under 3000 behaviour score, players are muted, maybe 2500 u mean?
reduced behaviour score average of player / reports? 

if you queue regions that are higher than 150 ping that you rarely / never play before, you might not find game due to server network need to download your data i think? e.g. i tried EU server with 350 ping for first time, took me 15 min search? and maybe time difference e.g. maybe 4 am in US time, so most players are sleeping.

also what i meant by search other region is that you should que only 1 region at a time, e.g. EU west try 10 min, if cant find then cancel queue, change to EU east. imagine you joined a game lobby, found 2-3 players but not a full 10 required, then it will take longer for you find game in that region if there are not enough players. 

i think sea server is most populated server should only take about 2-4 min.
